docs/Ahk2ExeDirectives.htm

```diff
@@ -0,0 +1,284 @@
+<tr><td width="15%">When</td><td width="85%">Leave blank to execute before any <a href=scripts.htm#mpress>Compression</a> is done. Otherwise set to a number to run after compression as follows:<ul style="list-style-type:none;"><li>0 - Only run when no compression is specified.</li><li>1 - Only run when MPRESS compression is specified.</li><li>2 - Only run when UPX compression is specified.</li></ul></td></tr>
+</table>
+<p><strong>Example 1:</strong> (To use these examples, first download <a href="https://github.com/AutoHotkey/Ahk2Exe/blob/master/BinMod.ahk">BinMod.ahk</a> and compile it according to the instructions in the downloaded script.)</p>
+<p>This example can be used to remove a reference to "AutoHotkey" in the generated .exe to disguise that it is a compiled AutoHotkey script:
+<pre>
+;@Ahk2Exe-PostExec "BinMod.exe" "%A_WorkFileName%"
+;@Ahk2Exe-Cont  "22.&gt;AUTOHOTKEY SCRIPT&lt;.DATA               "
+</pre></p>
+<p><strong>Example 2:</strong> This example will alter a UPX compressed .exe so that it can't be de-compressed with <code>UPX -d</code>:
+<pre>
+;@Ahk2Exe-PostExec "BinMod.exe" "%A_WorkFileName%"
+;@Ahk2Exe-Cont  "11.UPX." "1.UPX!.", 2
+</pre></p><p>There are other examples mentioned in the <a href="https://github.com/AutoHotkey/Ahk2Exe/blob/master/BinMod.ahk">BinMod.ahk</a> script.</p>
+</div>
+
+<div class="methodShort"><h3 id="SetMainIcon">SetMainIcon</h3>
+<p>Overrides the custom EXE icon used for compilation. (To change the other icons, see the <a href="#AddResource">AddResource</a> example.)</p>
+<pre class="Syntax">;@Ahk2Exe-SetMainIcon [IcoFile]</pre>
+<table class="info">
+<tr><td width="15%">IcoFile</td><td width="85%"><i>(Optional)</i> The icon file to use. If omitted, the default AutoHotkey icon is used.</td></tr>
+</table>
+</div>
+
+<div class="methodShort"><h3 id="SetProp">Set<em>Prop</em></h3>
+<p>Changes a property in the compiled executable's version information. Note that all properties are processed in alphabetical order, regardless of the order they are specified.</p>
+<pre class="Syntax">;@Ahk2Exe-Set<i>Prop</i> Value</pre>
+<table class="info">
+<tr><td width="15%">Prop</td><td width="85%">The name of the property to change. Must be one of those listed below.</td></tr>
+<tr><td width="15%">Value</td><td width="85%">The value to set the property to.</td></tr>
+</table>
+<p><strong>The following properties are supported by this directive:</strong></p>
+<table class="info">
+<tr><td width="15%">CompanyName</td><td width="85%">Changes the company name.</td></tr>
+<tr><td width="15%">Copyright</td><td width="85%">Changes the legal copyright information.</td></tr>
```

Comment:
Good work including all the information. I know this template is largely based on fincs' original page, but I think all the table widths can be optimized so they don't have to be specified on each line. As far as I know, the following is valid HTML4.01 and HTML5:
```
<table class="info">
<colgroup><col style="width:15%"><col style="width:80%"></colgroup>
<tr><td>Property Name</td><td width="85%">Some description here.</td></tr>
<tr><td>Property Name2</td><td width="85%">Some description2 here.</td></tr>
</table>
```
I can do this later on, if you prefer.

Reply:
I think just doing `<colgroup><col style="width:15%"></colgroup>` is likely sufficient too.

Reply:
I'm ok with you doing this if you think this change is worth while.
Cheers

Reply:
I've done some testing, including with IE11 in compat. mode IE=5 and it works fine. Go ahead and change these. :+1:

Assuming these are all 15 / 85 % (2 column) tables, then removing all the width style tags and adding `<colgroup><col style="width:15%"></colgroup>` to the beginning of each table will work just fine. It will be cleaner and easier to maintain.